src/pages/studyView/table/FixedHeaderTable.tsx

```diff
@@ -60,6 +54,11 @@ export default class FixedHeaderTable<T> extends React.Component<IFixedHeaderTab
         this.initDataStore();
     }
 
+    componentWillReceiveProps() {
+        // TODO do not re-initialize the whole store, just update the changed props within the store
+        this.initDataStore();
+    }
+
     @action
     initDataStore() {
```

Comment:
This function was tagged as `@action`, but it was not changing any observable properties.